src/App.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Route, Switch } from 'react-router';
-import { PrivateRoute } from './components/Shared/PrivateRoute';
 
+import './App.scss';
+import { PrivateRoute } from './components/Shared/PrivateRoute';
```

Comment:
Since we are only exporting one thing from that file lets not use the {} so we keep things consistent